cms/static/sass/_build-v1.scss

```diff
@@ -81,3 +81,7 @@
 
 @import 'developer'; // used for any developer-created scss that needs further polish/refactoring
 @import 'shame';     // used for any bad-form/orphaned scss
+
```

Comment:
I'd like to replace our main blue/green, but it will have larger implications. I don't want to shame this, because it needs to be a permanent change.